reporter/src/funTest/assets/npm-test-with-exclude-expected-NOTICE

```diff
@@ -5,57 +5,53 @@ This project contains or depends on third-party software components pursuant to
 Copyright (c) Felix Bohm
 (c) http://mathiasbynens.be/notes/javascript-encoding
 
-Copyright (c) <year> <owner> . All rights reserved.
-
 Redistribution and use in source and binary forms, with or without modification,
 are permitted provided that the following conditions are met:
 
```

Comment:
PR: as the ids of scancode are internal ones, would it make sense to move the LicenseRef text resource
underneath some "scancode" dir in order to indicate those are scancode specific?

Reply:
That would esp. make sense once we have a second source besides ScanCode for non-SPDX license texts, but I'd prefer to postpone this until then as I feel that's out of scope of this PR.

Reply:
Makes sense.